### PR TITLE
replace theme modal with style modal

### LINKED ISSUE #80: Unify the chart position & theme modal
We talked about this in the Discord.
Modal would be split into two like such the add candidates modal.

### PR DESCRIPTION
**Pull request by Fish1**
Preview:
Combined the theme menu and chart position button into a style modal.
image
image

Code:
Renamed the theme modal to the style modal, and all of the related components/stores.
The chart position buttons are toggles.

closes #80

### CODE REVIEW

File: apps/yapms/src/routes/app/[country]/[map]/[year]/+page.svelte [hunk 1]
```diff
@@ -19,8 +19,8 @@
 	import ChartBar from '$lib/components/chartbar/ChartBar.svelte';
 	import PresetColorsModal from '$lib/components/modals/presetcolorsmodal/PresetColorsModal.svelte';
 	import ChartTypeModal from '$lib/components/modals/charttypemodal/ChartTypeModal.svelte';
```
Comment (Flizzyclone) [inline apps/yapms/src/routes/app/[country]/[map]/[year]/+page.svelte:21]: Importing chart type modal as well here

**Review by Flizzyclone — APPROVED** [review]
LGTM

**Comment by Flizzyclone** [comment]
Oh, one thing though- the button styles you have for selected and unselected are inconsistent with other parts of the app. For example, mode selection:
image

vs:

image

May want to change this.

**Comment by Fish1** [comment]
Oh true, I'll change it to the blue/green style you've used.

**Pull request merged**